### PR TITLE
Fix issue #172 when remove ForbiddenSecurityIssues Insight

### PR DESCRIPTION
**Pull request by Jibbarth**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Fixed tickets | #172 

When ForbiddenSecurityIssues is removed in config, we caught an exception. 

This should fix it

### CODE REVIEW

**Comment by Jibbarth** [comment]
@olivernybroe looking at [changelog](https://github.com/nunomaduro/phpinsights/compare/v1.6.0...master) we have 2 new Insights, a composer.json update, more documentation, and this fix.

With new Insights, I think we should tag 1.7 if we follow semver. But maybe we should wait the new wordpress preset before tagging ?
WDYT ?

Anyway, thanks for the review :+1:

**Comment by olivernybroe** [comment]
Oh yeah, I guess 2 new insights is a lot for a small release 👍 

Let's wait for next change bigger change. Which hopefully is wordpress preset

File: src/Domain/Results.php [hunk 1]
```diff
@@ -102,8 +102,8 @@ public function getTotalSecurityIssues(): int
                 return count($insight->getDetails());
             }
         }
-
-        throw new \RuntimeException('This should not happen');
+        // ForbiddenSecurityInsight is not included
+        return 0;
```
Comment (nunomaduro) [inline src/Domain/Results.php]: Humm, what happens with the console output with this change?

Comment (Jibbarth) [reply src/Domain/Results.php]: Regarding https://github.com/nunomaduro/phpinsights/blob/master/src/Application/Console/Style.php#L188

The color is green and its display 
`0 security issues encountered`.

Should we remove this message when this Insight is removed @nunomaduro ?

Comment (nunomaduro) [reply src/Domain/Results.php]: Yeah, let's create an issue for this.

Comment (nunomaduro) [reply src/Domain/Results.php]: Done: https://github.com/nunomaduro/phpinsights/issues/181

**Pull request merged**